sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/OpenCoverReportParser.java

```diff
@@ -158,7 +158,7 @@ private void handleBranchPointTag(XmlParserHelper xmlParserHelper) {
         int visitCount = xmlParserHelper.getRequiredIntAttribute("vc");
 
         if (coveredFile.isIndexed()) {
-          coverage.add(new BranchPoint(coveredFile.indexedPath, line, offset, offsetEnd, path, visitCount));
+          coverage.add(new BranchPoint(coveredFile.indexedPath, line, offset, offsetEnd, path, visitCount, file.getPath()));
```

Comment:
Educational: is OpenCover the only tool that is using BranchPoint and doing branch analysis?

Reply:
The other tools do not use BranchPoints. If I understood Costin correctly, they do branch coverage in a more limited fashion.